package.json

```diff
@@ -36,6 +36,7 @@
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
     "jest": "^21.2.1",
+    "json-loader": "^0.5.7",
```

Comment:
Do we need the json-loader?  I know I mentioned it a few weeks back, but then I removed the "json" line from webpack.config.js, and it pulled in the json files without an issue.

Reply:
I got an error that was basically saying I need to have json-loader when I ran npm start after running npm run build.

Reply:
fair enough